packages/lesswrong/components/icons/SettingsButton.tsx

```diff
@@ -33,11 +34,11 @@ const SettingsButton = ({classes, className, onClick, showIcon=true, label=""}:
 }) => {
   if (label) {
     return <span className={classes.iconWithLabelGroup} onClick={onClick}>
-      {showIcon && <Settings className={classNames(classes.icon, classes.iconWithLabel, className)}/>}
+      {showIcon && <Sort className={classNames(classes.icon, classes.iconWithLabel, className)}/>}
       <span className={classes.label}>{ label }</span>
     </span>
   }
-  return <Settings className={classNames(classes.icon, className)} onClick={onClick}/>
+  return <Sort className={classNames(classes.icon, className)} onClick={onClick}/>
```

Comment:
I'm a bit concerned about this icon being too close to the one we use for shortform, there is a material UI icon called (confusingly) SwapVert which has the little arrows which might be better
<img width="930" alt="Screenshot 2023-01-12 at 17 49 38" src="https://user-images.githubusercontent.com/77623106/212141707-295bb77e-2190-482e-9cc0-619353538e7d.png">

Reply:
Oh brilliant! Will update to that, thanks